[docs] [Drivers and ORMs] update nodejs pages

aishwarya24 opened the pull request:
@netlify /preview/drivers-orms/nodejs/
https://deploy-preview-13007--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/nodejs/
Missing content - 

- Nodejs smart driver

priyanshi-yb commented on docs/content/preview/drivers-orms/nodejs/_index.md:
```diff
@@ -46,4 +52,9 @@ To develop Node.js applications for YugabyteDB, you need the following:
+## Next steps
+
+- Learn how to build NodeJS applications using [Sequelize](sequelize/).
+- Learn how to use [Prisma](../../integrations/prisma/) with YugabyteDB.
```
`Prisma` should point to the page `(prisma/)`.

priyanshi-yb commented on docs/content/preview/drivers-orms/nodejs/_index.md:
```diff
@@ -13,17 +12,24 @@ menu:
+
+| Project | Documentation and Guides | Example Apps |
+| :------ | :----------------------- | :----------- |
+| Sequelize | [Documentation](sequelize) <br /> [Hello World App](../../quick-start/build-apps/nodejs/ysql-sequelize/) | [Sequelize ORM App](https://github.com/yugabyte/orm-examples/tree/master/node/sequelize) |
```
The `Sequelize` row needs to be removed and all the links and examples which are there in this Sequelize row are for `Sequelize Yugabytedb`  which uses YugabyteDB specific package for sequelize as for now. `Sequelize` page in `Use an ORM` section explains the use with core sequelize with some limitations with YugabyteDB, so that is not required to be mentioned in this table.
CC: @ashetkar

priyanshi-yb commented:
For missing content - 

1. Created a [PR](https://github.com/yugabyte/yugabyte-db/pull/13062).
2. Smart-driver for node.js is not published yet. So, will add the documentation for that after that.
3. Refer to this [comment](https://github.com/yugabyte/yugabyte-db/pull/13007#discussion_r907178983).

aishwarya24 commented:
@priyanshi-yb . I have addressed your suggestions. I have a question. The code block is different for the node-postgres driver on the build an app page and the drivers and ORM page. Could you please let me know which one to follow , so that we can have the same at both places.

priyanshi-yb commented:
Thanks @aishwarya24 !
In that case, Use the build an app page's code block and modify the output and dependencies installation accordingly. Let me know if you have any questions here.
